plugins/module_utils/netbox_utils.py

```diff
@@ -443,7 +470,9 @@ def _convert_identical_keys(self, data):
             if data.get("form_factor"):
                 temp_dict["type"] = data["form_factor"]
         for key in data:
-            if key in CONVERT_KEYS:
+            if self.endpoint == 'power_panels' and key == 'rack_group':
+                temp_dict[key] = data[key]
+            elif key in CONVERT_KEYS:
```

Comment:
@FragmentedPacket: This workaround works but it is not very pretty.
I've found no other endpoint with a similar problem. It seems to be an inconsistency in the API from NetBox.

Do you have any suggestions on how to proceed regarding this?

Reply:
I think this is the way I would have done it at the moment anyways. We may want to update the unit tests to account for this.